### PR TITLE
webgpu: support linSpace operator

### PR DESCRIPTION
**Pull request by xhcao**
To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

- - -
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7119)

### CODE REVIEW

File: tfjs-backend-webgpu/src/kernels/LinSpace.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  const program = new LinSpaceProgram(num);
+  const uniformData =
+      [{type: 'float32', data: [start]}, {type: 'float32', data: [step]}];
+  return backend.runWebGPUProgram(program, [], 'float32', uniformData);
+}
+
+export const linSpaceConfig: KernelConfig = {
+  kernelName: LinSpace,
+  backendName: 'webgpu',
+  kernelFunc: linSpace as {} as KernelFunc
```
Comment (qjia7) [inline tfjs-backend-webgpu/src/kernels/LinSpace.ts:38]: nit: Per #7116, use `as unknown as KernelFunc`.

**Review by qjia7 — APPROVED** [review]
LGTM with one nit.

File: tfjs-backend-webgpu/src/kernels/LinSpace.ts (hunk 1 above)
Comment (xhcao) [reply tfjs-backend-webgpu/src/kernels/LinSpace.ts:38]: Could we submit a CL to change all kernels, not modify only one here?

Comment (xhcao) [reply tfjs-backend-webgpu/src/kernels/LinSpace.ts]: Done

**Review by gyagp — APPROVED** [review]

**Pull request merged**